docs/sources/tempo/traceql/_index.md

```diff
@@ -128,6 +128,18 @@ It's a convenient request to identify misconfigurations and leaks across product
 { resource.deployment.environment = "production" } && { resource.deployment.environment = "staging" }
 ```
 
+### Find traces with arrays
+
+TraceQL automatically queries data contained in arrays.
+Support for arrays is available in vParquet4 and on.
+
+If `span.foo` is an array and contains the value `bar`, then it will be found by this query.
+
+```
+{ span.foo = "bar" }
```

Comment:
can we also add an example to regex.

You can use regex to match multiple values of array `{span.http.request.header.Accept=~"application.*"}` and get all values of the array with `.*` RegEx `{span.http.request.header.Accept=~".*"}`

Reply:
Will do! Thank you for the suggestion.